### PR TITLE
Adding drones

### PR DESCRIPTION
**Pull request by pcabreracamara**
Hi,

I add some commercial drones interesting hacks

Regards,
Pedro

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,3 +181,12 @@ the table of content before making pull requests.
 - [Hello Barbie Initial Security Analysis](https://static1.squarespace.com/static/543effd8e4b095fba39dfe59/t/56a66d424bf1187ad34383b2/1453747529070/HelloBarbieSecurityAnalysis.pdf)
 - [Security researcher Ken Munro discovers vulnerability in Vivid Toy's talking Doll 'Cayla'](http://www.techworm.net/2015/01/vivid-toys-cayla-talking-doll-vulnerable-hacking-says-security-researcher.html)
 - [Data from connected CloudPets teddy bears leaked and ransomed, exposing kids' voice messages](https://www.troyhunt.com/data-from-connected-cloudpets-teddy-bears-leaked-and-ransomed-exposing-kids-voice-messages/)
+
+### Drones
+
+- [Parrot Drones Hijacking - RSA2018 Video, Pedro Cabrera, March 2018](https://youtu.be/66z-aXy_1Yo)
+- [Parrot Drones Hijacking - RSA2018 Slides, Pedro Cabrera, March 2018](https://www.rsaconference.com/writable/presentations/file_upload/mbs-w14-parrot-drones-hijacking.pdf)
```
Comment (nebgnahz) [inline README.md]: This is the same information as the above one. Can you combine them into a single entry?

You can probably add a hyperlink [slides](https://www.rsaconference.com/writable/presentations/file_upload/mbs-w14-parrot-drones-hijacking.pdf) at the end of the first line.

File: README.md [hunk 2]
```diff
@@ -180,3 +181,12 @@ the table of content before making pull requests.
...
+- [All your bebop drones still belong to us, drone hijacking, Pedro Cabrera, 2016](https://youtu.be/ra0nKHvaXnc)
```
Comment (nebgnahz) [inline README.md]: Can you sort these hacks by time?

**Review by nebgnahz — COMMENTED** [review]
Hey, thanks for the PR. Great work on discovering the vulnerabilities in drones and summarizing them! I have two comments about entry duplication and ordering. Please address them.

File: README.md (hunk 2 above)
Comment (pcabreracamara) [reply README.md]: Hi,

Now you can find the requested modifications applied; combined two links and ordered by time (descendant).

:)

Comment (nebgnahz) [reply README.md]: I think that is invalid Markdown...

screenshot 2018-11-03 22 20 24

Comment (pcabreracamara) [reply README.md]: Yep, now it's solved, sorry.

**Comment by nebgnahz** [comment]
LGTM. Merged. Thanks!

**Pull request merged**